core/src/main/scala/kafka/server/AbstractFetcherThread.scala

```diff
@@ -265,6 +265,7 @@ abstract class AbstractFetcherThread(name: String,
       Errors.forCode(leaderEpochOffset.errorCode) match {
         case Errors.NONE =>
           val offsetTruncationState = getOffsetTruncationState(tp, leaderEpochOffset)
+          info(s"Truncating partition $tp to leader epoch and offset $leaderEpochOffset")
```

Comment:
Would it be useful to include the offset from `offsetTruncationState` in this message since it is not necessarily the same as `leaderEpochOffset`?

Reply:
Good idea. I pushed up this change and I did the same thing with the hwm based truncation too.